### ISSUE TITLE
why limit to `data.table`?

### ISSUE THREAD

**Issue by timelyportfolio** [dcenergy/rflot#1]
@gjoneski, Nice job, and glad to have found you through your line note.

Do you plan to add support for data of class != to `data.table`?  I think for more widespread adoption `data.frame` will be required.    Was there some reason for choosing to exclusively support `data.table`?

**Comment by timelyportfolio** [comment]
At a minimum, it seems you could easily convert a `data.frame` to `data.table`, but this would still require the `data.table` package.

**Comment by timelyportfolio** [comment]
Can understand if you as the developer only want `data.table`, but as an experiment forked and made a [branch](https://github.com/timelyportfolio/rflot/tree/try-dataframe) where `data.frame` is available, but stopped before fully implementing in case you are not interested.

**Comment by gjoneski** [comment]
Hi there,

Thanks for your feedback.

When I started putting this package together, I envisioned some scenarios where data.table would make a difference from the perspective of both performance and syntaxical simplicity.  Looking at where the code has ended up - it's not really clear to me that I've used data.table in such a way so as to make gains in either direction.  I think your suggestion has merit!

I'll leave this issue open to make sure we revisit this.
